contracts/rewarders/SimpleRewarderPerSec.sol

```diff
@@ -48,6 +48,14 @@ interface IMasterChefJoe {
  * E.g. say you've allocated 100,000 XYZ to the JOE-XYZ farm over 30 days. Then you would need to transfer
  * 100,000 XYZ and set the block reward accordingly so it's fully distributed after 30 days.
  *
+ *
+ * Issue with previous version is that this can return 0 or be very inacurate with some tokens:
+ *      uint256 timeElapsed = block.timestamp.sub(pool.lastRewardTimestamp);
+ *      uint256 tokenReward = timeElapsed.mul(tokenPerSec);
+ *      accTokenPerShare = accTokenPerShare.add(
+ *          tokenReward.mul(ACC_TOKEN_PRECISION).div(lpSupply)
+ *      );
+ *  The goal with those changes is to prevent this, without any overflow too.
```

Comment:
```
 * Issue with the previous version is that this fraction, `tokenReward.mul(ACC_TOKEN_PRECISION).div(lpSupply)`, 
 * can return 0 or be very inacurate with some tokens:
 *      uint256 timeElapsed = block.timestamp.sub(pool.lastRewardTimestamp);
 *      uint256 tokenReward = timeElapsed.mul(tokenPerSec);
 *      accTokenPerShare = accTokenPerShare.add(
 *          tokenReward.mul(ACC_TOKEN_PRECISION).div(lpSupply)
 *      );
 *  The goal is to set ACC_TOKEN_PRECISION high enough to prevent this without causing overflow too.
 ```